app/models/laboratory.rb

```diff
@@ -56,7 +57,7 @@ def custom_details(fields, patient_identifiers)
     laboratory_details[:user_defined_id_statelocal] = patient_identifiers[:user_defined_id_statelocal]
     laboratory_details[:user_defined_id_cdc] = patient_identifiers[:user_defined_id_cdc]
     laboratory_details[:user_defined_id_nndss] = patient_identifiers[:user_defined_id_nndss]
-    laboratory_details[:lab_type] = lab_type || '' if fields.include?(:lab_type)
+    laboratory_details[:lab_type] = remove_formula_start(lab_type) || '' if fields.include?(:lab_type)
```

Comment:
are there `lab_types` in production other than the ones here? https://github.com/SaraAlert/SaraAlert/blob/master/app/helpers/validation_helper.rb#L113

if not, is this necessary since the laboratory validation should not allow lab types to start with formulas?

Reply:
Yeah I missed this. I believe that it could be removed here.

Reply:
I'm ready to approve after this is removed